tests/test_mgxs_library_distribcell/test_mgxs_library_distribcell.py

```diff
@@ -59,7 +59,7 @@ def _get_results(self, hash_output=False):
             for mgxs_type in avg_lib.mgxs_types:
                 mgxs = avg_lib.get_mgxs(domain, mgxs_type)
                 df = mgxs.get_pandas_dataframe()
-                outstr += df.to_string()
+                outstr += df.to_string() + '\n'
```

Comment:
I presume this is what changed the diff above, why is this necessary? Perhaps it makes the results cleaner.